common/lib/xmodule/xmodule/backcompat_module.py

```diff
@@ -60,6 +60,8 @@ def from_xml(cls, xml_data, system, id_generator):
 
 
 class SemanticSectionDescriptor(XModuleDescriptor):
+    resources_dir = None
```

Comment:
Would it make sense to set this in the `XModuleDescriptor` base class, since `XModules` don't use (generally) use the XBlock resource loading API?